src/resource_clients/store_collection.ts

```diff
@@ -0,0 +1,69 @@
+import ow from 'ow';
+import { ApiClientSubResourceOptions } from '../base/api_client';
+import { ResourceCollectionClient } from '../base/resource_collection_client';
+import { PaginatedList } from '../utils';
+import { ActorStats } from './actor';
+
+/**
+ * @hideconstructor
```

Comment:
FYI this has no effect on the generated docs, you need to put a `@hidden` jsdoc over the constructor itself (this problem is present in many other places in this repo)